CRM/Contribute/BAO/Contribution.php

```diff
@@ -2098,6 +2098,7 @@ public static function transitionComponents($params) {
     $contribution->find();
 
     $contribution->loadRelatedObjects($input, $ids);
```

Comment:
TransitionComponents is deprecated so easier to move this function here to load memberships rather than waste time on refactor